README.md

```diff
@@ -1318,13 +1318,56 @@ ingress:
 * Description: If you use an image that is not public. then you can create dockerconfigjson secrets on your cluster and
   reference them here.
 
+### Elastic
+
+#### `elastic.enabled`
+
+* Required: false
+* Default: `false`
+* Description: Enable or disable elastic search subsystem
+> [!IMPORTANT]  
+> Alfresco can only use one search subsystem at a time, if you still have SOLR enabled (which you -shouldn't- when enabling this option),
+> SOLR will take precedence in the alfresco configmap.
+
+#### `elastic.host`
+
+* Required: false
+* Default: `elasticsearch`
+* Description: Host (service) name of elastic.
+
+#### `elastic.port`
+
+* Required: false
+* Default: `9200`
+* Description: Port of the elastic service.
+
+#### `elastic.createIndexIfNotExists`
+
+* Required: false
+* Default: `true`
+* Description: When enabled, and if the elastic index does not exist, it will be created automatically.
+
+#### `global.elasticsearch.indexName`
```

Comment:
Are you *sure* you want to use a global setting for this?

It feels a bit weird to have this one setting as global, and all the rest namespaced properly.

This would also mean that all customer helm charts that pull in this chart will need to set `global.elasticsearch.indexName`.

Reply:
we can do here what we did for the re-indexing chart: have both a global and local setting with the global one undefined by default

Reply:
Done.